src/midi/midi_fluidsynth.cpp

```diff
@@ -63,6 +67,40 @@ void MidiHandlerFluidsynth::SetMixerLevel(const AudioFrame &desired_level) noexc
 	prescale_level.right = INT16_MAX * desired_level.right;
 }
 
+// Takes in the user's soundfont = configuration value consisting
+// of the SF2 filename followed by a optional scaling percentage.
+// This function returns the filename and percentage as a tuple.
+// If a percentage isn't provided, then it returns 'default_percent'.
+std::tuple<std::string, int> parse_sf_pref(const std::string &line,
+                                           const int default_percent)
+{
+	if (line.empty())
+		return std::make_tuple(line, default_percent);
```

Comment:
One last note: when returning a tuple, you can simply write:
```C++
    return {line, default_percent};
```
when building with -O2 or higher it does not matter, but otherwise, it will create slightly smaller code than `std::make_tuple`. And it looks cleaner :)

Reply:
That's some new syntax! Thanks @dreamer.
Much easier on the eyes.. will drop this in.